keyboards/zen/keymaps/333fred/rules.mk

```diff
@@ -0,0 +1,12 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = no
+NKRO_ENABLE = yes
+KEY_LOCK_ENABLE = yes
+TAP_DANCE_ENABLE = yes
+CONSOLE_ENABLE = no
+
+ifndef QUANTUM_DIR
```

Comment:
If you could remove this block?  It's a holdover from the old makefile system, and not needed anymore.